src/main/java/com/tabnine/inline/InlineCompletionCache.kt

```diff
@@ -0,0 +1,39 @@
+package com.tabnine.inline
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.util.Key
+import com.tabnine.prediction.TabNineCompletion
+import java.util.stream.Collectors
+
+class InlineCompletionCache {
+
+    companion object {
```

Comment:
it holds just the instance and the key. wdym?

Reply:
im not sure what are the differences, since this object is transparent to outside callers, but I know that usually when you want a singleton of a class you only wrap the `getInstance` method in a companion object, like so:

```kotlin
companion object {
        @JvmStatic
        val instance: YourClass
            get() = // instantiate your class
    }
```